ADBench/J_Errors.txt

```diff
@@ -0,0 +1,203 @@
+  Mismatch between files Manual/hand1_J_manual_eigen.txt and Finite/hand1_J_Finite_light.txt in Release/hand/complicated_big
+  Mismatch between files Theano/hand1_J_Theano.txt and Manual/hand1_J_manual_eigen.txt in Release/hand/complicated_big
+  Mismatch between files Finite/hand2_J_Finite_light.txt and ADOLC/hand2_J_ADOLC_light.txt in Release/hand/complicated_big
+  Mismatch between files Manual/hand2_J_manual_eigen.txt and Finite/hand2_J_Finite_light.txt in Release/hand/complicated_big
+  Mismatch between files Theano/hand2_J_Theano.txt and Manual/hand2_J_manual_eigen.txt in Release/hand/complicated_big
+  Mismatch between files Finite/hand3_J_Finite_light.txt and ADOLC/hand3_J_ADOLC_light.txt in Release/hand/complicated_big
+  Mismatch between files Manual/hand3_J_manual_eigen.txt and Finite/hand3_J_Finite_light.txt in Release/hand/complicated_big
+  Mismatch between files Theano/hand3_J_Theano.txt and Manual/hand3_J_manual_eigen.txt in Release/hand/complicated_big
+  Mismatch between files Finite/hand4_J_Finite_light.txt and ADOLC/hand4_J_ADOLC_light.txt in Release/hand/complicated_big
+  Mismatch between files Manual/hand4_J_manual_eigen.txt and Finite/hand4_J_Finite_light.txt in Release/hand/complicated_big
+  Mismatch between files Theano/hand4_J_Theano.txt and Manual/hand4_J_manual_eigen.txt in Release/hand/complicated_big
+  Mismatch between files Finite/hand5_J_Finite_light.txt and ADOLC/hand5_J_ADOLC_light.txt in Release/hand/complicated_big
+  Mismatch between files Manual/hand5_J_manual_eigen.txt and Finite/hand5_J_Finite_light.txt in Release/hand/complicated_big
+  Mismatch between files Theano/hand5_J_Theano.txt and Manual/hand5_J_manual_eigen.txt in Release/hand/complicated_big
+Finding files for graph: Debug/hand/simple_small
+Comparing files for graph: Debug/hand/simple_small
+  Mismatch between files ADOLC/hand1_J_ADOLC_light.txt and Adept/hand1_J_Adept_light.txt in Debug/hand/simple_small
+  Mismatch between files ADOLC/hand2_J_ADOLC_light.txt and Adept/hand2_J_Adept_light.txt in Debug/hand/simple_small
+  Mismatch between files Finite/hand2_J_Finite_light.txt and ADOLC/hand2_J_ADOLC_light.txt in Debug/hand/simple_small
+  Mismatch between files Manual/hand2_J_manual_eigen.txt and Finite/hand2_J_Finite_light.txt in Debug/hand/simple_small
+  Mismatch between files ADOLC/hand3_J_ADOLC_light.txt and Adept/hand3_J_Adept_light.txt in Debug/hand/simple_small
+  Mismatch between files ADOLC/hand4_J_ADOLC_light.txt and Adept/hand4_J_Adept_light.txt in Debug/hand/simple_small
+  Mismatch between files Finite/hand4_J_Finite_light.txt and ADOLC/hand4_J_ADOLC_light.txt in Debug/hand/simple_small
+  Mismatch between files Manual/hand4_J_manual_eigen.txt and Finite/hand4_J_Finite_light.txt in Debug/hand/simple_small
+  Mismatch between files ADOLC/hand5_J_ADOLC_light.txt and Adept/hand5_J_Adept_light.txt in Debug/hand/simple_small
```

Comment:
I think we only want to compare finite and Manual, and then Manual and the others.
Finite should agree with Manual to about 4 sigfigs, and then manual to the others to say 8 sigfigs.